### DISCUSSION TITLE
How do I simulate network latency for all actions?

### DISCUSSION THREAD

**Discussion by bastinald** [livewire/livewire#3643] [1. Help]
How can I simulate network latency on local host for every single livewire action?

Say add 1 second to every action so I can see what it will be like on production?

I have been adding sleep(1) to my methods, but there’s gonna be a global way I hope.

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

**Comment by lennardv2** [comment]
Hey @bastinald 

Is this for performance testing on bad connections? You can use the throttling option in chrome developer tools.

https://css-tricks.com/throttling-the-network/
